docker-files/Dockerfile.ubi.min

```diff
@@ -19,6 +19,8 @@ RUN microdnf update -y \
 	e2fsprogs \
 	xfsprogs \
 	nfs-utils \
+    nfs4-acl-tools \
```

Comment:
these 2 lines need formatting

Reply:
In the IDE, formatting is proper for these two lines.